web/server/www/scripts/codecheckerviewer/CommentView.js

```diff
@@ -108,7 +107,7 @@ function (declare, dom, style, topic, Memory, Observable, ConfirmDialog,
         content   : 'Are you sure you want to delete this?',
         onExecute : function () {
           try {
-            CC_SERVICE.removeComment(that.cId);
+            CC_SERVICE.removeComment(that.commentData.id);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['CC_SERVICE' is not defined. (no-undef)](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=3374942)